exporter.go

```diff
@@ -819,6 +821,23 @@ func (e *Exporter) extractTile38Metrics(ch chan<- prometheus.Metric, c redis.Con
 	}
 }
 
+func (e *Exporter) extractConnectedClientMetrics(ch chan<- prometheus.Metric, c redis.Conn) {
+	if reply, err := redis.String(doRedisCmd(c, "CLIENT", "LIST")); err == nil {
+		clients := strings.Split(reply, "\n")
+		
+		var clientsMap = map[string]int{}
+		for i := 0; i < len(clients) - 1; i++ {
+			clientInfo := strings.Split(clients[i], " ")
+			name := strings.Split(clientInfo[3], "=")[1]
```

Comment:
After calling Split()`, you should check that the resulting array has enough elements before accessing e.g. `[1]`, see e.g. https://github.com/oliver006/redis_exporter/blob/9aef08c64d755d66f92dbc894cbdd3be5a28bd01/exporter.go#L379-L380 for a code example.